stable/datacube-dashboard/values.yaml

```diff
@@ -4,7 +4,7 @@
 replicaCount: 1
 image:
   registry: docker.io
-  repository: opendatacube/dashboard
+  repository: opendatacube/datacube-explorer
```

Comment:
There is no docker `datacube-explorer` it should still be `dashboard`

Reply:
Updated to `dashboard` docker name